core/src/main/java/org/jacorb/poa/RequestProcessor.java

```diff
@@ -97,18 +100,31 @@
      */
     private static int count = 0;
 
-    private final static Set<String> specialOperations;
+    private enum SpecialOps { NONE, IS_A, INTERFACE, NON_EXISTENT, GET_POLICY, SET_POLICY, GET_COMPONENT, REPO_ID};
```

Comment:
Could we not do something like http://www.javapractices.com/topic/TopicAction.do?Id=1 which would simplify this I think ?